docs/book/src/topics/custom-dns.md

```diff
@@ -0,0 +1,37 @@
+# Custom Private DNS Zone Name
+
+It is possible to set the Private DNS Zone Name in the NetworkSpec to change the Private DNS zone name by default the dns zone name is `${CLUSTER_NAME}.capz.io`
+
+*This feature is enable only if the `apiServerLB.type` is `Internal`*
```

Comment:
```suggestion
*This feature is enabled only if the `apiServerLB.type` is `Internal`*
```

Reply:
I apply all of them, thanks for the review :smile: